---
layout: default
---
<section id="banner-projects">
</section>

<section id="main" class="wrapper banner">
  <!--  <header class="major">
	<h2>Projects</h2>
  </header> -->
  <div class="container">

    <h2>Ongoing Projects</h2>

    <img id="iba" class="image portrait" src="{{ site.baseurl }}/images/IBA.jpeg" alt="iba" width="200" />
    <strong>Insect Biome Atlas (IBA)</strong><br /><br />
    <p><a href="https://www.insectbiomeatlas.org">Insect Biome
Atlas (IBA)</a> is an international collaborative effort to describe in
detail the insect faunas of two biologically and geologically very
different countries: Sweden and Madagascar. The project, one of the
largest ongoing insect biodiversity surveys, addresses key questions
about the insect diversity: How are insect species distributed across
habitats, sites and seasons? What are the key environmental factors
shaping insect diversity?<br/><br/>

Collection phase lasted a whole year (2019) in each country and was
done by means of Malaise traps: 200 in Sweden and 50 in
Madagascar. Summing up, the IBA collected more than 8000 insect
community samples. Several other types of samples and ecological
measurements were also collected at the trap sites to gather a full
understanding of the ecological roles of the organisms that comprise
the insect biome in these countries. Analysis of the samples include
identification of all insects and the organisms they interact with,
such as pathogens as well as symbiotic fungi and bacteria - this is
achieved by using novel DNA techniques. The results and material
collected during the IBA project will be useful for scientists
interested in systematics, taxonomy and insect ecology and evolution
for many years to come.<br/><br/>

In Sweden the Malaise traps were managed by over 100 volunteers, which
makes this project one of the largest citizen science projects to take
place in Scandinavia.<br/><br/>
 
IBA consortium brings together researchers from many countries and
  institutes including <a href="https://www.nrm.se/">Natural History Museum in Stockholm</a>, <a href="https://su.se">Stockholm University</a>, <a href="https://www.slu.se/en/">Swedish University of Agricultural Sciences</a>, Uppsala, <a href="https://www.scilifelab.se/">SciLifeLab</a>, KTH Royal Institute of Technology Stockholm, <a href="https://www.madagascarbio.org/">Madagascar Biodiversity Center</a>, <a href="https://www.uj.edu.pl/en_GB/">Jagiellonian University</a>, Krakow, Poland. <br/><br/>
Involved Researchers: <a href="{{ site.baseurl }}/people.html#adreia">Andreia Miraldo</a>, <a href="{{ site.baseurl }}/people.html#ela">Elzbieta Iwaszkiewicz</a></p>

<div class="hr-section"></div>

	   <img id="phyppl" class="image portrait" src="{{ site.baseurl }}/images/FinBio.pdf" alt="FinBio" width="200" />
    <strong>Finance to Revive Biodiversity (FinBio)</strong><br /><br />
    <p> Biodiversity loss is unravelling the web of life that supports all people, societies, economies, including the financial system. MISTRA Finance to Revive Biodiversity (<a href="http://finbio.org">FinBio</a>) will provide cutting-edge research to connect the financial system and biodiversity. 
	The program is hosted by Stockholm Resilience Centre at Stockholm University and is a partnership between academic and financial actors. The mission is to support the financial sector’s capacity to contribute to a nature-positive economy, enhancing the resilience of our planet by reversing the loss of nature and biodiversity. <br/><br/>

The Ronquist lab will contribute by for example testing new environmental-DNA techniques to track and map biodiversity change in ways that produce open source data to inform financial decisions. <br/><br/>
	  
     Involved Researchers: <a href="{{ site.baseurl }}/people.html#emma">Emma Granqvist</a></p>


<div class="hr-section"></div>



 <img id="phyppl" class="image portrait" src="{{ site.baseurl }}/images/yggdrasil022.webp" alt="IBA" width="200" />
    <strong>Phylogenetic Probabilistic Programming Languages (PhyPPL)</strong><br /><br />
    <p>PhyPPL is a Marie-Curie-financed project aimed to achieve the first large-scale application of probabilistic programming in phylogenetics. It is closely linked to the development of the <a href="http://treeppl.org/">TreePPL probabilistic programming language for phylogenetics.</a> The project's goals are to develop evolutionary models taking advantage of the advanced inference methods provided by sequential Monte Carlo algorithms and implement them in the new <a href="http://treeppl.org">TreePPL</a> programming language.<br/>
    The project is supported by a grant from the European Union’s Horizon 2020 research and innovation program under the Marie Skłodowska-Curie grant agreement PhyPPL No 898120 to Viktor Senderov.</br>
    <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#viktor">Viktor Senderov</a>, <a href="{{ site.baseurl }}/people.html#emma">Emma Granqvist</a></p>


<div class="hr-section"></div>



 <img class="image portrait" src="{{ site.baseurl }}/images/eggs.JPG" alt="wasp phylogeny" width="200" />
    <strong>Cynipoid Phylogenomics</strong><br /><br />
   
    <p>
      We study gall-wasps to try to understand the genetic basis of their biology. Gall-wasps (or Cynipids) are a family of parasitic wasps that attack oaks and other plants. They are notable for their habit of manipulating their host by forcing it to produce a complex structure where the larvae are fed and protected. However, some cynipids are unable to initiate the process and usually live in the gall of other insects (they are hence termed the "inquilines"). In order to identify genes responsible for gall-induction and clarify its exact mechanism, we compare the genomes of gall-inducers and inquilines. We also work on improving our understanding of both the phylogeny of the group and the life-history of the various species, as this kind of knowledge is essential to the form.
      <br/>
      <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#erik">Erik Gobbo</a>
    </p>

    <div class="hr-section"></div>

    
    
    <h2>Previous Projects</h2>
    

    <img class="image portrait" src="{{ site.baseurl }}/images/megaselia-hansonix.jpg" alt="Megaselia hansonix" width="200" />
    <strong>Dipteran taxonomy and morphology</strong><br /><br />
   
    <p>
      The projects were centered on exploration of the large, "open-ended" genus Megaselia (Diptera: Phoridae). Large, diverse groups like Megaselia can be excellent tools for investigating ecosystems but the sheer size and diversity of these groups pose a number of logistic challenges. The projects focused on the identification of Megaselia in Sweden (including the description of species new to science), the phylogeny of Megaselia, streamlining species discovery using massive parallel sequencing (MPS), and recovering life history data of the clade using MPS.
      <br/>
      <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#emily">Emily Hartop</a>
    </p>

    <div class="hr-section"></div>



   
    <img class="image portrait" src="{{ site.baseurl }}/images/stoneflies.jpg" alt="stoneflies" width="200" />
    <strong>Automatic organism identification using machine learning</strong><br /><br />
    <p>Rapid and reliable identifications of organisms are important in many contexts, from the detection of disease vectors and invasive species to the sorting of material from biodiversity inventories. Because of the shortage of adequate expertise, there has long been an interest in developing automated systems for such tasks. Today, sophisticated AI models can help us to automate some of these processes. Although not always at the human expert-level, these models are doing it much faster and can be applied on scales unmanageable for manual processing.<br /><br />

    Our research mainly focused, but was not limited to, the use of state-of-the-art machine learning in computer vision, convolutional neural networks (CNNs). Our research addressed questions such as:<br /><br />
    
    - automated taxon identifications using effective feature transfer (<a href="https://academic.oup.com/sysbio/article/68/6/876/5368535">paper</a>)<br />

    - tackling tasks which are deemed impossible to solve by humans (submitted)<br/>
      
    - engaging citizen scientist to effectively collect data (read more about our citizen science project which engaged 15 000 swedish pupils - <a href="https://forskarfredag.se/forskarfredags-massexperiment/nyckelpigeforsoket/">website</a> )<br/>
      
      - the use of genetic data for improving image based automated identification tools (in progress)<br/>
      
      - the use of CNNs for inferring phylogeny (planned)<br/>
      
      - development of ready-to-deploy solutions - for example identification of wildlife from camera trap images (<a href="https://arxiv.org/abs/2008.07828">paper</a>)<br/>
      
      <br/><br/>
      Involved Researchers: <a href="{{ site.baseurl }}/people.html#miroslav">Miroslav Valan</a>, <a href="{{ site.baseurl }}/people.html#allison">Allison Hsiang</a>
    </p>

    <div class="hr-section"></div>
    <hr>

  
    <img class="image portrait" src="{{ site.baseurl }}/images/MTsample.jpg" alt="metabarcoding" width="200"/>
    <strong>Metabarcoding of insects for accelerated taxonomic discovery</strong><br /><br />
   
    <p>
      This project, part of the European
      project <a href="http://big4-project.eu">BIG4 consortium</a>
      (Horizon 2020 framework - Marie Curie innovative training
      network), was focused on the optimization of protocols for
      metabarcoding of insects from bulk samples or environmental DNA
      (eDNA).  <br /> <br > Our research stressed the importance of
      the marker choice as one of the key factors in the metabarcoding
      workflow, in terms of maximising taxonomic coverage and
      resolution, as well as which marker is more adequate for
      different types of samples.  <br /> <br > We also focused on
      laboratory methods for non-destructive DNA extraction, trying to
      find the best protocol that effectively extracts DNA while
      minimizing the effect on the structural integrity of the
      specimens, so that their morphology can be examined by
      taxonomists after obtaining the sequencing results.
      <br /> <br > This synergy between metabarcoding and taxonomy can
      speed up the discovery and description of new species.
      <br /> <br > Involved Researchers: <a href="https://metagusano.github.io/">Daniel Marquina</a>
    </p>

    <div class="hr-section"></div>


    <img class="image portrait" src="{{ site.baseurl }}/images/morphology-model.png" alt="modeling" width="200" />
    <strong>Probabilistic modelling of morphological evolution</strong><br /><br />
   
    <p>
      Morphology represents an important source of data for understanding evolutionary history, particularly for fossil taxa, which remain the only means by which we can directly observe the history of life. Modern statistical phylogenetic inference methods using Bayesian and maximum likelihood frameworks require probabilistic models of evolution, such as the Generalized Time-Reversible (GTR) model of DNA evolution. However, existing probabilistic models of morphological evolution are relatively simple, and fail to capture the complexity of morphological evolution – in the context of phylogenetic inference, their use is analogous to using only the Jukes-Cantor model for phylogenetic inference using nucleotide data. We are thus developing new probabilistic models of morphological evolution to address this gap. In particular, we are modelling correlation between traits, using both discrete-state continuous-time Markov models and multivariate normal models to detect correlation structure in phenomic datasets.
      <br /><br />
      Involved Researchers: <a href="{{ site.baseurl }}/people.html#allison">Allison Hsiang</a>
    </p>

    <div class="hr-section"></div>



    <img class="image portrait" src="{{ site.baseurl }}/images/host-parasite.jpg" alt="wasp phylogeny" width="200" />
    <strong>Evolution of butterfly-plant associations</strong><br /><br />
   
    <p>
      In collaboration with Michael Landis (Washington University in St. Louis), we developed a Bayesian approach for inferring the history of association between butterflies and their host plants. We describe the evolution of the repertoire of hosts that a butterfly uses as a continuous-time Markov process of gain and loss of discrete hosts, where gains are influenced by the phylogenetic distance between hosts. This method will allows us to reconstruct ancestral networks and investigate more realistic problems that involve ecological interactions among many taxa. Related papers: Braga et al. 2020 Syst Biol, Braga et al. 2021 Ecol Let.
      <br/>
      <br />Involved Researchers: <a href="{{ site.baseurl }}/people.html#mariana">Mariana Braga</a>
    </p>

    <div class="hr-section"></div>


 <!-- <header class="major">
      <h2>Associated Projects</h2>
    </header>-->
    

    <img class="image portrait" src="{{ site.baseurl }}/images/knowledge-base.png" alt="Semantic knowledge bases" width="200" />
    <strong>Semantic Knowledge Bases for Biodiversity</strong>  <br /><br />
     <p>
      A knowledge base (KB) about biodiversity called <a href="http://openbiodiv.net" target="_blank">OpenBiodiv</a>. The KB consists of semantic apps (expert apps) that answer specific questions that biodiversity scientists have, as well as general access services running on top of a semantic graph database. The database is powered by the OpenBiodiv Ontology <http://openbiodiv.net/openbiodivo-20171103> and populated by information extracted from scholarly articles.
        <br/>
        <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#viktor">Viktor Senderov</a>
    </p>

    <div class="hr-section"></div>

  </div>

  <div class="hr-section"></div>

</section>
