<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

<html>

<head>
<title>pub</title>
</head>

<body>

<!-- This document was automatically generated with bibtex2html 1.99
     (see http://www.lri.fr/~filliatr/bibtex2html/),
     with the following command:
     bibtex2html -labelname pub.bib  -->


<table>

<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1093/sysbio/syad054">1</a>]
</td>
<td class="bibtexitem">
Chi Zhang, Fredrik Ronquist, and Tanja Stadler.
 Skyline Fossilized Birth–Death Model is Robust to Violations of
  Sampling Assumptions in Total-Evidence Dating.
 <em>Systematic Biology</em>, 72(6):1316--1336, 08 2023.
[&nbsp;<a href="pub_bib.html#10.1093/sysbio/syad054">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syad054">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://academic.oup.com/sysbio/article-pdf/72/6/1316/56917945/syad054.pdf">arXiv</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syad054">Url</a>&nbsp;]
<blockquote><font size="-1">
Several total-evidence dating studies under the fossilized birth–death (FBD) model have produced very old age estimates, which are not supported by the fossil record. This phenomenon has been termed “deep root attraction (DRA).” For two specific data sets, involving divergence time estimation for the early radiations of ants, bees, and wasps (Hymenoptera) and of placental mammals (Eutheria), it has been shown that the DRA effect can be greatly reduced by accommodating the fact that extant species in these trees have been sampled to maximize diversity, so-called diversified sampling. Unfortunately, current methods to accommodate diversified sampling only consider the extreme case where it is possible to identify a cut-off time such that all splits occurring before this time are represented in the sampled tree but none of the younger splits. In reality, the sampling bias is rarely this extreme and may be difficult to model properly. Similar modeling challenges apply to the sampling of the fossil record. This raises the question of whether it is possible to find dating methods that are more robust to sampling biases. Here, we show that the skyline FBD (SFBD) process, where the diversification and fossil-sampling rates can vary over time in a piecewise fashion, provides age estimates that are more robust to inadequacies in the modeling of the sampling process and less sensitive to DRA effects. In the SFBD model we consider, rates in different time intervals are either considered to be independent and identically distributed or assumed to be autocorrelated following an Ornstein–Uhlenbeck (OU) process. Through simulations and reanalyses of Hymenoptera and Eutheria data, we show that both variants of the SFBD model unify age estimates under random and diversified sampling assumptions. The SFBD model can resolve DRA by absorbing the deviations from the sampling assumptions into the inferred dynamics of the diversification process over time. Although this means that the inferred diversification dynamics must be interpreted with caution, taking sampling biases into account, we conclude that the SFBD model represents the most robust approach currently available for addressing DRA in total-evidence dating.
</font></blockquote>

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="https://doi.org/10.1111/syen.12611">2</a>]
</td>
<td class="bibtexitem">
Jack Hearn, Erik Gobbo, José&nbsp;Luis Nieves-Aldrey, Antoine Branca, James&nbsp;A.
  Nicholls, Georgios Koutsovoulos, Nicolas Lartillot, Graham&nbsp;N. Stone, and
  Fredrik Ronquist.
 Phylogenomic analysis of protein-coding genes resolves complex gall
  wasp relationships.
 <em>Systematic Entomology</em>, 49(1):110--137, 2024.
[&nbsp;<a href="pub_bib.html#https://doi.org/10.1111/syen.12611">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/syen.12611">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://resjournals.onlinelibrary.wiley.com/doi/pdf/10.1111/syen.12611">arXiv</a>&nbsp;| 
<a href="https://resjournals.onlinelibrary.wiley.com/doi/abs/10.1111/syen.12611">Url</a>&nbsp;]
<blockquote><font size="-1">
Abstract Gall wasps (Hymenoptera: Cynipidae) comprise 13 distinct tribes whose interrelationships remain incompletely understood. Recent analyses of ultra-conserved elements (UCEs) represent the first attempt at resolving these relationships using phylogenomics. Here, we present the first analysis based on protein-coding sequences from genome and transcriptome assemblies. Unlike UCEs, these data allow more sophisticated substitution models, which can potentially resolve issues with long-branch attraction. We include data for 37 cynipoid species, including two tribes missing in the UCE analysis: Aylacini (s. str.) and Qwaqwaiini. Our results confirm the UCE result that Cynipidae are not monophyletic. Specifically, the Paraulacini and Diplolepidini + Pediaspidini fall outside a core clade (Cynipidae s. str.), which is more closely related to the insect-parasitic Figitidae, and this result is robust to the exclusion of long-branch taxa that could mislead the analysis. Given this, we here divide the Cynipidae into three families: the Paraulacidae stat. prom., Diplolepididae stat. prom. and Cynipidae (s. str.). Our results suggest that the Eschatocerini are the sister group of the remaining Cynipidae (s. str.). Within the Cynipidae (s. str.), the Aylacini (s. str.) are more closely related to oak gall wasps (Cynipini) and some of their inquilines (Ceroptresini) than to other herb gallers (Aulacideini and Phanacidini), and the Qwaqwaiini likely form a clade together with Synergini (s. str.) and Rhoophilini. Several alternative scenarios for the evolution of cynipid life histories are compatible with the relationships suggested by our analysis, but all are complex and require multiple shifts among parasitoids, inquilines and gall inducers.
</font></blockquote>
<blockquote><font size="-1">
Keywords: gall inducers, gall wasps, inquilines, life history transitions, parasitoids, phylogenomics, protein-coding genes
</font></blockquote>

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="doi:10.1098/rspb.2024.0090">3</a>]
</td>
<td class="bibtexitem">
Laura J.&nbsp;A. van Dijk, Brian&nbsp;L. Fisher, Andreia Miraldo, Robert&nbsp;M. Goodsell,
  Elzbieta Iwaszkiewicz-Eggebrecht, Dimby Raharinjanahary, Eric&nbsp;Tsiriniaina
  Rajoelison, Piotr Łukasik, Anders&nbsp;F. Andersson, Fredrik Ronquist, Tomas
  Roslin, and Ayco J.&nbsp;M. Tack.
 Temperature and water availability drive insect seasonality across a
  temperate and a tropical region.
 <em>Proceedings of the Royal Society B: Biological Sciences</em>,
  291(2025):20240090, 2024.
[&nbsp;<a href="pub_bib.html#doi:10.1098/rspb.2024.0090">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1098/rspb.2024.0090">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://royalsocietypublishing.org/doi/pdf/10.1098/rspb.2024.0090">arXiv</a>&nbsp;| 
<a href="https://royalsocietypublishing.org/doi/abs/10.1098/rspb.2024.0090">Url</a>&nbsp;]
<blockquote><font size="-1">
 The more insects there are, the more food there is for insectivores and the higher the likelihood for insect-associated ecosystem services. Yet, we lack insights into the drivers of insect biomass over space and seasons, for both tropical and temperate zones. We used 245 Malaise traps, managed by 191 volunteers and park guards, to characterize year-round flying insect biomass in a temperate (Sweden) and a tropical (Madagascar) country. Surprisingly, we found that local insect biomass was similar across zones. In Sweden, local insect biomass increased with accumulated heat and varied across habitats, while biomass in Madagascar was unrelated to the environmental predictors measured. Drivers behind seasonality partly converged: In both countries, the seasonality of insect biomass differed between warmer and colder sites, and wetter and drier sites. In Sweden, short-term deviations from expected season-specific biomass were explained by week-to-week fluctuations in accumulated heat, rainfall and soil moisture, whereas in Madagascar, weeks with higher soil moisture had higher insect biomass. Overall, our study identifies key drivers of the seasonal distribution of flying insect biomass in a temperate and a tropical climate. This knowledge is key to understanding the spatial and seasonal availability of insects—as well as predicting future scenarios of insect biomass change. 
</font></blockquote>

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1371/journal.pone.0286272">4</a>]
</td>
<td class="bibtexitem">
Elzbieta Iwaszkiewicz-Eggebrecht, Piotr Lukasik, Mateusz Buczek, Junchen
  Deng, Emily&nbsp;A. Hartop, Harald Havn&aring;s, Monika Prus-Frankowska, Carina&nbsp;R.
  Ugarph, Paulina Viteri, Anders&nbsp;F. Andersson, Tomas Roslin, Ayco J.&nbsp;M. Tack,
  Fredrik Ronquist, and Andreia Miraldo.
 Favis: Fast and versatile protocol for non-destructive metabarcoding
  of bulk insect samples.
 <em>PLOS ONE</em>, 18(7):1--13, 07 2023.
[&nbsp;<a href="pub_bib.html#10.1371/journal.pone.0286272">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1371/journal.pone.0286272">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1371/journal.pone.0286272">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1093/sysbio/syac033">5</a>]
</td>
<td class="bibtexitem">
Emily Hartop, Amrita Srivathsan, Fredrik Ronquist, and Rudolf Meier.
 Towards Large-Scale Integrative Taxonomy (LIT): Resolving the Data
  Conundrum for Dark Taxa.
 <em>Systematic Biology</em>, 71(6):1404--1422, 05 2022.
[&nbsp;<a href="pub_bib.html#10.1093/sysbio/syac033">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syac033">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://academic.oup.com/sysbio/article-pdf/71/6/1404/46465829/syac033.pdf">arXiv</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syac033">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1093/sysbio/syad045">6</a>]
</td>
<td class="bibtexitem">
Jakub Truszkowski, Allison Perrigo, David Broman, Fredrik Ronquist, and
  Alexandre Antonelli.
 Online tree expansion could help solve the problem of scalability in
  Bayesian phylogenetics.
 <em>Systematic Biology</em>, page syad045, 07 2023.
[&nbsp;<a href="pub_bib.html#10.1093/sysbio/syad045">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syad045">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://academic.oup.com/sysbio/advance-article-pdf/doi/10.1093/sysbio/syad045/51305994/syad045.pdf">arXiv</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syad045">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1007/978-3-031-30044-8_20">7</a>]
</td>
<td class="bibtexitem">
Daniel Lund&eacute;n, Gizem &Ccedil;aylak, Fredrik Ronquist, and David Broman.
 Automatic alignment in higher-order probabilistic programming
  languages.
 In Thomas Wies, editor, <em>Programming Languages and Systems</em>,
  pages 535--563, Cham, 2023. Springer Nature Switzerland.
[&nbsp;<a href="pub_bib.html#10.1007/978-3-031-30044-8_20">bib</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="https://doi.org/10.1111/2041-210X.14073">8</a>]
</td>
<td class="bibtexitem">
Elzbieta Iwaszkiewicz-Eggebrecht, Emma Granqvist, Mateusz Buczek, Monika Prus,
  Jan Kudlicka, Tomas Roslin, Ayco J.&nbsp;M. Tack, Anders&nbsp;F. Andersson, Andreia
  Miraldo, Fredrik Ronquist, and Piotr Lukasik.
 Optimizing insect metabarcoding using replicated mock communities.
 <em>Methods in Ecology and Evolution</em>, 14(4):1130--1146, 2023.
[&nbsp;<a href="pub_bib.html#https://doi.org/10.1111/2041-210X.14073">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/2041-210X.14073">DOI</a>&nbsp;| 
<a href="http://arxiv.org/abs/https://besjournals.onlinelibrary.wiley.com/doi/pdf/10.1111/2041-210X.14073">arXiv</a>&nbsp;| 
<a href="https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14073">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="10.1007/978-3-030-99336-8_2">9</a>]
</td>
<td class="bibtexitem">
Daniel Lund&eacute;n, Joey &Ouml;hman, Jan Kudlicka, Viktor Senderov, Fredrik
  Ronquist, and David Broman.
 Compiling universal probabilistic programming languages with
  efficient parallel sequential monte carlo inference.
 In <em>Programming Languages and Systems</em>, pages 29--56, Cham, 2022.
  Springer International Publishing.
[&nbsp;<a href="pub_bib.html#10.1007/978-3-030-99336-8_2">bib</a>&nbsp;| 
<a href="https://doi.org/10.1007/978-3-030-99336-8_2">DOI</a>&nbsp;| 
<a href="https://link.springer.com/chapter/10.1007/978-3-030-99336-8_2">Url</a>&nbsp;| 
<a href="https://link.springer.com/content/pdf/10.1007/978-3-030-99336-8.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="braga2021phylogenetic">10</a>]
</td>
<td class="bibtexitem">
Mariana&nbsp;P. Braga, Niklas Janz, S&ouml;ren Nylin, Fredrik Ronquist, and
  Michael&nbsp;J. Landis.
 Phylogenetic reconstruction of ancestral ecological networks through
  time for pierid butterflies and their host plants.
 <em>Ecology Letters</em>, 24:2134 --2145, 2021.
[&nbsp;<a href="pub_bib.html#braga2021phylogenetic">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/ele.13842">DOI</a>&nbsp;| 
<a href="https://www.onlinelibrary.wiley.com/doi/abs/10.1111/ele.13842">Url</a>&nbsp;| 
<a href="https://www.onlinelibrary.wiley.com/doi/pdf/10.1111/ele.13842">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="valan2021awakening">11</a>]
</td>
<td class="bibtexitem">
Miroslav Valan, Dominik Vondr&aacute;&#x010D;ek, and Fredrik Ronquist.
 Awakening a taxonomist's third eye: exploring the utility of computer
  vision and deep learning in insect systematics.
 <em>Systematic Entomology</em>, 46:757--766, 2021.
[&nbsp;<a href="pub_bib.html#valan2021awakening">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/syen.12492">DOI</a>&nbsp;| 
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/syen.12492">Url</a>&nbsp;| 
<a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/syen.12492">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="Hartop1544804">12</a>]
</td>
<td class="bibtexitem">
Emily Hartop.
 <em>A multi-faceted approach to a "dark taxon" : The hyperdiverse
  and poorly known scuttle flies (Diptera: Phoridae)</em>.
 PhD thesis, Station Linn&eacute;, 2021.
[&nbsp;<a href="pub_bib.html#Hartop1544804">bib</a>&nbsp;| 
<a href="http://urn.kb.se/resolve?urn=urn%3Anbn%3Ase%3Asu%3Adiva-192276">Url</a>&nbsp;| 
<a href="http://www.diva-portal.org/smash/get/diva2:1544804/FULLTEXT01.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="hartop2021large">13</a>]
</td>
<td class="bibtexitem">
Emily Hartop, Amrita Srivathsan, Fredrik Ronquist, and Rudolf Meier.
 Large-scale Integrative Taxonomy (LIT): resolving the data conundrum
  for dark taxa.
 <em>bioRxiv</em>, 2021.
[&nbsp;<a href="pub_bib.html#hartop2021large">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1101/2021.04.13.439467">DOI</a>&nbsp;| 
<a href="https://www.biorxiv.org/content/early/2021/05/09/2021.04.13.439467">Url</a>&nbsp;| 
<a href="https://www.biorxiv.org/content/early/2021/05/09/2021.04.13.439467.full.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="valan2021automated">14</a>]
</td>
<td class="bibtexitem">
Miroslav Valan.
 <em>Automated image-based taxon identification using deep learning
  and citizen-science contributions</em>.
 PhD thesis, Stockholm University, Department of Zoology, 2021.
[&nbsp;<a href="pub_bib.html#valan2021automated">bib</a>&nbsp;| 
<a href="http://urn.kb.se/resolve?urn=urn%3Anbn%3Ase%3Asu%3Adiva-189460">Url</a>&nbsp;| 
<a href="http://www.diva-portal.org/smash/get/diva2:1522250/FULLTEXT01.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="ronquist2021universal">15</a>]
</td>
<td class="bibtexitem">
Fredrik Ronquist, Jan Kudlicka, Viktor Senderov, Johannes Borgstr&ouml;m,
  Nicolas Lartillot, Daniel Lund&eacute;n, Lawrence&nbsp;D. Murray, Thomas&nbsp;B.
  Sch&ouml;n, and David Broman.
 Universal probabilistic programming offers a powerful approach to
  statistical phylogenetics.
 <em>Communications Biology</em>, 4:244, 2021.
[&nbsp;<a href="pub_bib.html#ronquist2021universal">bib</a>&nbsp;| 
<a href="https://doi.org/10.1038/s42003-021-01753-7">DOI</a>&nbsp;| 
<a href="https://www.nature.com/articles/s42003-021-01753-7">Url</a>&nbsp;| 
<a href="https://www.nature.com/articles/s42003-021-01753-7.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="arribas2021connecting">16</a>]
</td>
<td class="bibtexitem">
Paula Arribas, Carmelo And&uacute;jar, Martin&nbsp;I. Bidartondo, Kristine Bohmann,
  &Eacute;ric Coissac, Simon Creer, Jeremy&nbsp;R. deWaard, Vasco Elbrecht, Gentile&nbsp;F.
  Ficetola, Marta Goberna, Susan Kennedy, Henrik Krehenwinkel, Florian Leese,
  Vojtech Novotny, Fredrik Ronquist, Douglas&nbsp;W. Yu, Lucie Zinger, Thomas&nbsp;J.
  Creedy, Emmanouil Meramveliotakis, V&iacute;ctor Noguerales, Isaac Overcast,
  H&eacute;l&egrave;ne Morlon, Alfried&nbsp;P. Vogler, Anna Papadopoulou, and Brent&nbsp;C.
  Emerson.
 Connecting high-throughput biodiversity inventories: Opportunities
  for a site-based genomic framework for global integration and synthesis.
 <em>Molecular Ecology</em>, 30:1120--1135, 2021.
[&nbsp;<a href="pub_bib.html#arribas2021connecting">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/mec.15797">DOI</a>&nbsp;| 
<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.15797">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="hartop2021scuttling">17</a>]
</td>
<td class="bibtexitem">
Emily Hartop, Sibylle H&auml;ggqvist, Sven&nbsp;Olof Ulefors, and Fredrik Ronquist.
 Scuttling towards monophyly: phylogeny of the mega-diverse genus
  <i>Megaselia</i> (Diptera: Phoridae).
 <em>Systematic Entomology</em>, 46:71--82, 2021.
[&nbsp;<a href="pub_bib.html#hartop2021scuttling">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1111/syen.12448">DOI</a>&nbsp;| 
<a href="http://doi.org/10.1111/syen.12448">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="marquina2021effect">18</a>]
</td>
<td class="bibtexitem">
Daniel Marquina, Mateusz Buczek, Fredrik Ronquist, and Piotr Lukasik.
 The effect of ethanol concentration on the morphological and
  molecular preservation of insects for biodiversity studies.
 <em>PeerJ</em>, 9:e10799, 2021.
[&nbsp;<a href="pub_bib.html#marquina2021effect">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.7717/peerj.10799">DOI</a>&nbsp;| 
<a href="https://doi.org/10.7717/peerj.10799">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="braga2020bayesian">19</a>]
</td>
<td class="bibtexitem">
Mariana&nbsp;P Braga, Michael&nbsp;J Landis, S&ouml;ren Nylin, Niklas Janz, and Fredrik
  Ronquist.
 Bayesian inference of ancestral host-parasite interactions under a
  phylogenetic model of host repertoire evolution.
 <em>Systematic Biology</em>, 69:1149--1162, 2020.
[&nbsp;<a href="pub_bib.html#braga2020bayesian">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syaa019">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syaa019">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="ronquist2020completing">20</a>]
</td>
<td class="bibtexitem">
Fredrik Ronquist, Mattias Forshage, Sibylle H&auml;ggqvist, Dave Karlsson,
  Rasmus Hovm&ouml;ller, Johannes Bergsten, Kevin Holston, Tom Britton, Johan
  Abenius, Bengt Andersson, et&nbsp;al.
 Completing Linnaeus's inventory of the Swedish insect fauna: only
  5,000 species left?
 <em>PLOS ONE</em>, 15(3):e0228561, 2020.
[&nbsp;<a href="pub_bib.html#ronquist2020completing">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1371/journal.pone.0228561">DOI</a>&nbsp;| 
<a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0228561">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="kudlicka2020probabilistic">21</a>]
</td>
<td class="bibtexitem">
Jan Kudlicka, Lawrence&nbsp;M Murray, Fredrik Ronquist, and Thomas&nbsp;B Sch&ouml;n.
 Probabilistic programming for birth-death models of evolution using
  an alive particle filter with delayed sampling.
 In <em>Uncertainty in Artificial Intelligence</em>, pages 679--689.
  PMLR, 2020.
[&nbsp;<a href="pub_bib.html#kudlicka2020probabilistic">bib</a>&nbsp;| 
<a href="http://auai.org/uai2019/proceedings/papers/235.pdf">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="zhang2020using">22</a>]
</td>
<td class="bibtexitem">
Chi Zhang, John&nbsp;P Huelsenbeck, and Fredrik Ronquist.
 Using parsimony-guided tree proposals to accelerate convergence in
  Bayesian phylogenetic inference.
 <em>Systematic Biology</em>, 69:1016--1032, 2020.
[&nbsp;<a href="pub_bib.html#zhang2020using">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syaa002">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syaa002">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="karlsson2020data">23</a>]
</td>
<td class="bibtexitem">
Dave Karlsson, Mattias Forshage, Kevin Holston, and Fredrik Ronquist.
 The data of the Swedish Malaise Trap Project, a countrywide
  inventory of Sweden's insect fauna.
 <em>Biodiversity Data Journal</em>, 8:e56286, 2020.
[&nbsp;<a href="pub_bib.html#karlsson2020data">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.3897/BDJ.8.e56286">DOI</a>&nbsp;| 
<a href="https://doi.org/10.3897/BDJ.8.e56286">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="karlsson2020swedish">24</a>]
</td>
<td class="bibtexitem">
Dave Karlsson, Emily Hartop, Mattias Forshage, Mathias Jaschhof, and Fredrik
  Ronquist.
 The Swedish Malaise trap project: a 15 year retrospective on a
  countrywide insect inventory.
 <em>Biodiversity Data Journal</em>, 8:e47255, 2020.
[&nbsp;<a href="pub_bib.html#karlsson2020swedish">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.3897/BDJ.8.e47255">DOI</a>&nbsp;| 
<a href="https://doi.org/10.3897/BDJ.8.e47255">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="marquina2020thesis">25</a>]
</td>
<td class="bibtexitem">
Daniel Marquina&nbsp;Hern&aacute;ndez.
 <em>Debugging metabarcoding for insect biodiversity studies</em>.
 PhD thesis, Stockholm University, Department of Zoology, 2020.
[&nbsp;<a href="pub_bib.html#marquina2020thesis">bib</a>&nbsp;| 
<a href="http://urn.kb.se/resolve?urn=urn%3Anbn%3Ase%3Asu%3Adiva-180293">Url</a>&nbsp;| 
<a href="http://www.diva-portal.org/smash/get/diva2:1417107/FULLTEXT01.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="ronquist2020probabilistic">26</a>]
</td>
<td class="bibtexitem">
Fredrik Ronquist, Jan Kudlicka, Viktor Senderov, Johannes Borgstr&ouml;m,
  Nicolas Lartillot, Daniel Lund&eacute;n, Lawrence Murray, Thomas&nbsp;B. Sch&ouml;n,
  and David Broman.
 Universal probabilistic programming offers a powerful approach to
  statistical phylogenetics.
 <em>bioRxiv</em>, 2020.
[&nbsp;<a href="pub_bib.html#ronquist2020probabilistic">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1101/2020.06.16.154443">DOI</a>&nbsp;| 
<a href="https://www.biorxiv.org/content/early/2020/12/10/2020.06.16.154443">Url</a>&nbsp;| 
<a href="https://www.biorxiv.org/content/early/2020/12/10/2020.06.16.154443.full.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="gobbo2020inquilines">27</a>]
</td>
<td class="bibtexitem">
Erik Gobbo, Nicolas Lartillot, Jack Hearn, Graham&nbsp;N Stone, Yoshihisa Abe,
  Christopher&nbsp;W Wheat, Tatsuya Ide, and Fredirik Ronquist.
 From inquilines to gall inducers: Genomic signature of a life-style
  transition in <i>Synergus</i> gall wasps.
 <em>Genome biology and evolution</em>, 12:2060--2073, 2020.
[&nbsp;<a href="pub_bib.html#gobbo2020inquilines">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/gbe/evaa204">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1093/gbe/evaa204">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="glockler2019dina">28</a>]
</td>
<td class="bibtexitem">
Falko Gl&ouml;ckler, James Macklin, Fredrik Ronquist, and Jana Hoffmann.
 DINA: Open Source and Open Services-A modern approach for natural
  history collection management systems and research.
 <em>Biodiversity Information Science and Standards</em>, 2019.
[&nbsp;<a href="pub_bib.html#glockler2019dina">bib</a>&nbsp;| 
<a href="https://doi.org/10.3897/biss.3.38059">DOI</a>&nbsp;| 
<a href="https://biss.pensoft.net/article/38059/list/9/">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="marquina2019new">29</a>]
</td>
<td class="bibtexitem">
Daniel Marquina, Anders&nbsp;F Andersson, and Fredrik Ronquist.
 New mitochondrial primers for metabarcoding of insects, designed and
  evaluated using <i>in silico</i> methods.
 <em>Molecular Ecology Resources</em>, 19(1):90--104, 2019.
[&nbsp;<a href="pub_bib.html#marquina2019new">bib</a>&nbsp;| 
<a href="https://doi.org/10.1111/1755-0998.12942">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1111/1755-0998.12942">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="marquina2019establishing">30</a>]
</td>
<td class="bibtexitem">
Daniel Marquina, Rodrigo Esparza-Salas, Tomas Roslin, and Fredrik Ronquist.
 Establishing arthropod community composition using metabarcoding:
  Surprising inconsistencies between soil samples and preservative ethanol and
  homogenate from Malaise trap catches.
 <em>Molecular ecology resources</em>, 19(6):1516--1530, 2019.
[&nbsp;<a href="pub_bib.html#marquina2019establishing">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1111/1755-0998.13071">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1111/1755-0998.13071">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="valan2019automated">31</a>]
</td>
<td class="bibtexitem">
Miroslav Valan, Karoly Makonyi, Atsuto Maki, Dominik Vondr&aacute;&#x010D;ek, and
  Fredrik Ronquist.
 Automated taxonomic identification of insects with expert-level
  accuracy using effective feature transfer from convolutional networks.
 <em>Systematic Biology</em>, 68(6):876--895, 2019.
[&nbsp;<a href="pub_bib.html#valan2019automated">bib</a>&nbsp;| 
<a href="http://dx.doi.org/10.1093/sysbio/syz014">DOI</a>&nbsp;| 
<a href="https://doi.org/10.1093/sysbio/syz014">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="braga2019evolution">32</a>]
</td>
<td class="bibtexitem">
Mariana&nbsp;Pires Braga.
 <em>Evolution of host repertoires and the diversification of
  butterflies</em>.
 PhD thesis, Stockholm University, Department of Zoology, 2019.
[&nbsp;<a href="pub_bib.html#braga2019evolution">bib</a>&nbsp;| 
<a href="http://urn.kb.se/resolve?urn=urn%3Anbn%3Ase%3Asu%3Adiva-163179">Url</a>&nbsp;| 
<a href="http://www.diva-portal.org/smash/get/diva2:1271841/FULLTEXT01.pdf">Pdf</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="lunden2018automatic">33</a>]
</td>
<td class="bibtexitem">
Daniel Lund&eacute;n, David Broman, Fredrik Ronquist, and Lawrence&nbsp;M Murray.
 Automatic alignment of sequential Monte Carlo inference in
  higher-order probabilistic programs.
 <em>arXiv preprint arXiv:1812.07439</em>, 2018.
[&nbsp;<a href="pub_bib.html#lunden2018automatic">bib</a>&nbsp;| 
<a href="https://arxiv.org/abs/1812.07439">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="macklin2018general">34</a>]
</td>
<td class="bibtexitem">
James Macklin, Markus Englund, Falko Gl&ouml;ckler, Mikko Heikkinen, Jana
  Hoffmann, Keith Newton, and Fredrick Ronquist.
 General introduction to DINA.
 <em>Biodiversity Information Science and Standards</em>, 2018.
[&nbsp;<a href="pub_bib.html#macklin2018general">bib</a>&nbsp;| 
<a href="https://doi.org/10.3897/biss.2.25646">DOI</a>&nbsp;| 
<a href="https://biss.pensoft.net/article/25646">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="ronquist2018life">35</a>]
</td>
<td class="bibtexitem">
Fredrik Ronquist, Johan Nylander, Hege V&aring;rdal, and Jos&eacute;&nbsp;Luis
  Nieves-Aldrey.
 Life history of <i>Parnips</i> and the evolutionary origin of
  gall wasps.
 <em>Journal of Hymenoptera Research</em>, 65:91--110, 2018.
[&nbsp;<a href="pub_bib.html#ronquist2018life">bib</a>&nbsp;| 
<a href="https://doi.org/10.3897/jhr.65.24115">DOI</a>&nbsp;| 
<a href="https://jhr.pensoft.net/articles.php?id=24115">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="hovmoller2017streckkodning">36</a>]
</td>
<td class="bibtexitem">
Rasmus Hovm&ouml;ller, Mattias Forshage, and Fredrik Ronquist.
 Streckkodning av den svenska floran och
  faunan--f&ouml;ruts&auml;ttningar och utmaningar.
 <em>PM fr&aring;n Naturhistoriska riksmuseet 2017: 1. Naturhistoriska
  riksmuseets sm&aring;skriftserie</em>, 2017.
[&nbsp;<a href="pub_bib.html#hovmoller2017streckkodning">bib</a>&nbsp;| 
<a href="https://www.nrm.se/download/18.2a85cf3215e50a71056eac46/1511363009228/Streckkodningsrapport_slutversion_2017-11-21.pdf">Url</a>&nbsp;]

</td>
</tr>


<tr valign="top">
<td align="right" class="bibtexnumber">
[<a name="macklin2017dina">37</a>]
</td>
<td class="bibtexitem">
James Macklin, Falko Gl&ouml;ckler, Jana Hoffmann, Fredrik Ronquist, Stefan
  Daume, and Elspeth Haston.
 DINA: Open Source and Open Services-A Modern Approach for
  Sustainable Natural History Collection Management Systems.
 <em>Biodiversity Information Science and Standards</em>, 2017.
[&nbsp;<a href="pub_bib.html#macklin2017dina">bib</a>&nbsp;| 
<a href="https://doi.org/10.3897/tdwgproceedings.1.20216">DOI</a>&nbsp;| 
<a href="https://biss.pensoft.net/article/20216/">Url</a>&nbsp;| 
<a href="https://biss.pensoft.net/article/20216/download/pdf/">Pdf</a>&nbsp;]

</td>
</tr>
</table><hr><p><em>This file was generated by
<a href="http://www.lri.fr/~filliatr/bibtex2html/">bibtex2html</a> 1.99.</em></p>
</body>
</html>
